src/app/dts/template-editor/template-editor.component.ts

```diff
@@ -218,7 +218,7 @@ export class TemplateEditorComponent extends UnsubscribeOnDestroyAdapter impleme
         this.statusMessage = 'Template saved';
       },
       (error) => {
-        console.log(`Save failed: $(error)`);
+        console.error(`Save failed: ${error}`);
```

Comment:
I think this syntax would give you a lot more information:
`console.error('Save failed', error);`

Reply:
Implemented the recommend change, and verified the unit test "should handle an error when saving a template" logs an informative message to the console.